src/formatDate.js

```diff
@@ -50,7 +50,29 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dateToArr = date.split(fromFormat[3]);
+
+  const obj = {};
+
+  const newFormat = [];
+
+  for (let i = 0; i < 3; i++) {
+    obj[fromFormat[i]] = dateToArr[i];
+  }
+
+  for (let i = 0; i < 3; i++) {
+    if (toFormat[i] in obj) {
+      newFormat.push(obj[toFormat[i]]);
+    } else if (toFormat[i] === 'YY') {
+      newFormat.push(obj['YYYY'].slice(2));
+    } else if (toFormat[i] === 'YYYY' & obj['YY'] < 30) {
+      newFormat.push(20 + obj['YY']);
```

Comment:
Use string interpolation for concatenating numbers and strings. Replace '20 + obj['YY']' with '`20${obj['YY']}`'.